### PR TITLE
adding vs 2.2 preview 1 to tip block

### PR DESCRIPTION
**Pull request by mairaw**

### CODE REVIEW

File: docs/core/windows-prerequisites.md [hunk 1]
```diff
@@ -118,11 +118,14 @@ To develop .NET Core 1.x apps in Visual Studio, [download and install Visual Stu
...
 ---
 
+<a name="vs-mapping"></a>
+
 > [!TIP]
 > To verify your Visual Studio 2017 version:
 >
 > * On the **Help** menu, choose **About Microsoft Visual Studio**.
 > * In the **About Microsoft Visual Studio** dialog, verify the version number.
+>   * For .NET Core 2.2 Preview 1 apps, Visual Studio 2017 version 15.9 Preview 1 or higher.
```
Comment (Andrew-MSFT) [inline docs/core/windows-prerequisites.md]: Question: would it be clearer to phrase this as "Visual Studio 2017 version 15.9 (currently in Preview)" to clarify that it's in the Preview channel?

**Comment by Andrew-MSFT** [comment]
Overall looks good, one question around the wording

**Review by AndrewBrianHall — APPROVED** [review]
LGTM

**Review by rpetrusha — APPROVED** [review]
LGTM, @mairaw. You can merge when you're ready.

**Pull request merged**